packages/components/select-v2/__tests__/select.test.ts

```diff
@@ -1517,6 +1517,35 @@ describe('Select', () => {
     expect(wrapper.findAll('.el-tag').length).toBe(0)
   })
 
+  it('tag list should be synchronized when the model value is the same', async () => {
+    const wrapper = _mount(
+      `<el-select
+        :model-value="[1]"
+        :options="[
+          {
+            value: 1,
+            label: 1,
+          },
+          {
+            value: 2,
+            label: 2,
+          },
+        ]"
+        clearable
+        multiple
+      />`
+    )
+    await wrapper.find(`.${WRAPPER_CLASS_NAME}`).trigger('click')
+
+    clickClearButton(wrapper)
+    const option = document.querySelector(`.${OPTION_ITEM_CLASS_NAME}`)
+    option.click()
+    await nextTick()
+    await nextTick()
+
+    expect(wrapper.findAll('.el-tag').length).toBe(1)
```

Comment:
Two nextTick look a bit strange, using `vi.waitUnit` seems to be better.
```suggestion
    const tags = await vi.waitUnit(() => wrapper.findAll('.el-tag'))
    expect(tags.length).toBe(1)
```

Reply:
You're right. Make the same correction in similar case.